test/metric_learn_test.py

```diff
@@ -72,6 +74,37 @@ def test_iris(self):
       self.assertLess(csep, 0.25)
 
 
+def test_convergence_simple_example(capsys):
```

Comment:
Where does capsys get passed in from?

Reply:
I think it should be automatically found by pytest (as one of the integrated fixtures), and runned when running `test_convergence_simple_example`. I verified doing -v (verbose) when doing pytest and these tests are correctly passing when they should (and failing when modifying the error message)